django/db/backends/sqlite3/features.py

```diff
@@ -38,6 +38,9 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     # Is "ALTER TABLE ... RENAME COLUMN" supported?
     can_alter_table_rename_column = Database.sqlite_version_info >= (3, 25, 0)
     supports_parentheses_in_compound = False
+    # Deferred constraint checks can be emulated on SQLite < 3.20 but not in a
+    # reasonably performant way.
+    can_defer_constraint_checks = Database.version_info >= (3, 20, 0)
```

Comment:
Is this supposed to be `sqlite_version_info` rather than `version_info`? I guess other usages in this file might be incorrect also.